www/templates/customer-balance.html

```diff
@@ -0,0 +1,4 @@
+<div>
+  <p>Balance: {{customer.balance}} rCredits</p>
+  <p>Rewards: {{customer.rewards}} rCredits</p>
```

Comment:
We would sometimes say "$12.34r", but in this case I think we would just
use "$12.34". The "r" is only necessary when distinguishing from USD.

On Tue, Dec 15, 2015 at 10:43 AM, Tom Smyth <notifications@github.com>
wrote:

> In www/templates/customer-balance.html
> <https://github.com/sassafrastech/rcredits-mobile/pull/4#discussion_r47652710>
> :
>
> > @@ -0,0 +1,4 @@
> > +<div>
> > +  <p>Balance: {{customer.balance}} rCredits</p>
> > +  <p>Rewards: {{customer.rewards}} rCredits</p>
>
> I believe the way to display this is "12.34r"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sassafrastech/rcredits-mobile/pull/4/files#r47652710>.
>